components/layout/Nav.js

```diff
@@ -44,11 +56,20 @@ const Nav = () => {
 const StyledNav = styled.nav`
   margin: 0;
   padding: 0;
-  display: flex;
-  justify-self: end;
-  font-size: 2rem;
+  section {
+    display: flex;
+    justify-content: flex-end;
+    font-size: 2rem;
+  }
+  .loggedOut {
+    @media (max-width: 750px) {
+      justify-content: flex-start;
```

Comment:
Where is the `display: flex` to use this?

Reply:
Why do you need `justify-content: flex-start;` for an anchor (`<a>`) element?
Can't you just use `text-align`?

Reply:
yes!